src/main/java/datatypes/Task.java

```diff
@@ -0,0 +1,39 @@
+package datatypes;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        setDescription(description);
+        this.isDone = false;
+    }
+
+    /**
+     * Returns the status of the current task
+     * The default status is a cross
+     * If the task is done it will return a tick
+     *
+     * @return returns a tick or a cross
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
```

Comment:
I'm not sure if "\u2713" and "\u2718" is very readable? I need to search for this meaning. Or maybe add some comments beside it